examples/bq-email-exports/terraform/payload.txt

```diff
@@ -0,0 +1,12 @@
+{
+    "project_id": "bq-email-exports",
+    "dataset_id": "bq_exports",
+    "table_name": "bq_export_",
+    "bucket_name": "bq_email_exports",
+    "query_bucket": "query_bucket",
```

Comment:
what is the value of putting SQL in GCS? 
Would be better to have the query as part of this config IMO
Similar to below you should just expose more BQ Query job to the user in this payload.

Namely:

-   "preserveNulls": boolean,
-   "allowLargeResults": boolean,
-   "useQueryCache": boolean,
-   "flattenResults": boolean,
-   "maximumBillingTier": integer,
-   "maximumBytesBilled": string,
-   "useLegacySql": boolean,

Reply:
I had originally put it in GCS because in the case that the user had a huge query, the formatting was messy. But since it just adds pre-work for the user, I agree it'll be better to have in the payload itself (with the query job configs)

Reply:
limit is 12MB for query length in BQ service anyway
https://cloud.google.com/bigquery/quotas#query_jobs
I think we'd be fine to keep it in the payload

Reply:
maximumBillingTier and preserveNulls are now deprecated, but I have included all the rest of the configs as a nested JSON block.